scripts/rnnlm/show_word_features.py

```diff
@@ -12,7 +12,7 @@
 # encoding as well as other encodings such as gbk, as long as the spaces are
```

Comment:
If you are going to switch to utf-8 you should remove these comments which explained why we were using latin-1.